sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1160,7 +1167,7 @@ object SQLConf {
     .intConf
     .createWithDefault(4096)
 
-  val PARQUET_FIELD_ID_WRITE_ENABLED =
+   val PARQUET_FIELD_ID_WRITE_ENABLED =
```

Comment:
? Is there a reason for this kind of change. It seems to have multiple instances, @parthchandra .